### DISCUSSION TITLE
neo4j after scan

### DISCUSSION THREAD

**Discussion by YehonatanEliyahu** [blacklanternsecurity/bbot#1527] [General]
let's say I performed a task like subdomain Enum and forgot the flag -om neo4j
is there a way to add the scan result to neo4j using bbot ?

**Comment by TheTechromancer** [comment]
Sadly this isn't trivial. If you're good with python and cypher, you might be able to ingest `output.json`.

**Comment by iukea1** [comment]
> let's say I performed a task like subdomain Enum and forgot the flag -om neo4j
> is there a way to add the scan result to neo4j using bbot ? 

I recommend you 
Credit python script that cleans up the Json output just a little. 


Then using the events that are generated by bbot craft that into a schema. 


I just did all events for everything so I never had to do it again
